securedrop/journalist_app/api2/events.py

```diff
@@ -135,3 +128,160 @@ def handle_reply_sent(event: Event) -> EventResult:
             sources={source.uuid: source},
             items={reply.uuid: reply},
         )
+
+    @staticmethod
+    def handle_source_deleted(event: Event) -> EventResult:
+        from journalist_app.api2 import json_version  # cyclic import
```

Comment:
Nit: Why not move `json_version()` to `shared.py`?  I guess it's not strictly speaking shared with the v1 API, but creating a separate `utils.py` just for this feels like overkill.  :-)